### PR TITLE
Fixed import loop cause by version.py

### PR DESCRIPTION
**Pull request by PatwinchIR**
## What does this PR do?

Updated a new way to import version by reading `version.json`.

## Why is this change being made?

To avoid an import-loop from the previous way of reading `VERSION`.

## How was this tested? How can the reviewer verify your testing?

This is tested by `nosetests` and entry point test `krux-boto-test`.
Also tested using `python setup.py sdist` to check the tarball included `version.json`

## Where should the reviewer start?

As stated above.

## Completion checklist

- [X] The pull request has been appropriately labelled according to
  our [conventions](https://kruxdigital.jira.com/wiki/display/EN/Changes+and+Peer+Review)

### CODE REVIEW

File: krux_boto/cli.py [hunk 1]
```diff
@@ -21,16 +23,17 @@
...
 
-from krux_boto.version import VERSION
-
 
 class Application(krux.cli.Application):
     # XXX: Usually, a VERSION constant should be set in __init__.py and be imported.
     #      However, krux-boto adds some basic classes to __init__.py and importing VERSION constant here
-    #      causes a dependency circle. Thus, set VERSION constant in version.py and import it
-    _VERSION = VERSION
+    #      causes a dependency circle. Thus, set VERSION constant in version.json and import it
```
Comment (hanpeter) [inline krux_boto/cli.py]: This comment should go with the code.

File: krux_boto/cli.py [hunk 2]
```diff
@@ -21,16 +23,17 @@
...
 
     def __init__(self, name=NAME, *args, **kwargs):
+        _VERSION_PATH = path.join(path.dirname(path.dirname(__file__)), 'version.json')
```
Comment (hanpeter) [inline krux_boto/cli.py]: Let's also comment on why there is two `path.dirname()` here.

File: setup.py [hunk 3]
```diff
@@ -12,20 +12,19 @@
...
 from setuptools import setup, find_packages
+from os import path
+import json
 
-#
-# Internal libraries
-#
-# We use the version to construct the DOWNLOAD_URL.
-from krux_boto.version import VERSION
+_VERSION_PATH = path.join(path.dirname(__file__), 'version.json')
```
Comment (hanpeter) [inline setup.py]: Please comment why this is needed.

File: krux_boto/cli.py (hunk 1 above)
Comment (PatwinchIR) [inline krux_boto/cli.py]: I think this comment should still be here because it's still true for this case.

Comment (PatwinchIR) [inline krux_boto/cli.py]: Maybe I can move it down under `def __init__`?

**Comment by hanpeter** [comment]
test this please

**Pull request merged**